Makefile

```diff
@@ -321,7 +321,6 @@ endif
 
 purge: purge-doc purge-dev user-uninstall
 	for FILE in ${R2BINS} ; do rm -f "${DESTDIR}${BINDIR}/$$FILE" ; done
-	rm -f "${DESTDIR}${BINDIR}/ragg2-cc"
```

Comment:
this is a purge. we want to remove it, because people installing older versions will have it

Reply:
please remove that line and ill merge